### ISSUE TITLE
Undefined reference to xdr_control when building raidz_test cmd 

### ISSUE THREAD

**Issue by SenH** [zfsonlinux/zfs#5922]
### System information

Type                                | Version/Name
  ---                                  |     --- 
Distribution Name       | Ubuntu
Distribution Version    | 12.04.5 LTS
Linux Kernel                 | 3.13.0-110-generic
Architecture                 | x86_64
ZFS Version                  | 0.7.0-rc3_154_g7b0dc2a31
SPL Version                  | 0.7.0-rc3_7_gbf8abea

### Describe the problem you're observing

Error after executing `make deb` 
```
Making all in raidz_test
make[5]: Entering directory `/tmp/zfs-build-sen-gWWLLFyc/BUILD/zfs-0.7.0/cmd/raidz_test'
  CC     raidz_test.o
  CC     raidz_bench.o
  CCLD   raidz_test
/lib64/libnvpair.so.1: undefined reference to `xdr_control'
collect2: ld returned 1 exit status
```

### Fixed with following patch

```
diff --git a/cmd/raidz_test/Makefile.am b/cmd/raidz_test/Makefile.am
index 23ad08ad9..15edb2b1e 100644
--- a/cmd/raidz_test/Makefile.am
+++ b/cmd/raidz_test/Makefile.am
@@ -15,6 +15,7 @@ raidz_test_SOURCES = \
        raidz_bench.c
 
 raidz_test_LDADD = \
+       $(top_builddir)/lib/libnvpair/libnvpair.la \
        $(top_builddir)/lib/libuutil/libuutil.la \
        $(top_builddir)/lib/libzpool/libzpool.la
 
```

**Comment by dinatale2** [comment]
@SenH Can you open a pull request for this? Thanks!

**Comment by SenH** [comment]
@dinatale2 https://github.com/SenH/zfs/pull/1

**Comment by behlendorf** [comment]
Fix from #5933 merged.
